.stylelintrc

```diff
@@ -1,5 +1,22 @@
 {
-  "processors": ["stylelint-processor-styled-components"],
+  "processors": [
```

Comment:
Need to add this because they changed the plugin internals in `v1.5.2`
Changes [here](https://github.com/styled-components/stylelint-processor-styled-components/pull/229/files#diff-b40ab9239a020bd6273bbb1134dedbb6R6)